editor.html

```diff
@@ -348,15 +348,15 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
                         <div class="buttons_menu_item"><span id="mp-ver">MicroPython Version:</span> <script>document.write(UPY_VERSION);</script></div>
                     </div>
                 </div>
-                <div id="small-icons" class="vbox">
+                <div id="small-icons-left" class="vbox small-icons">
                     <a aria-label="Zoom in" class="roundbutton holder zoomer action" tabindex="90" id="command-zoom-in" href="#" title="Zoom in">
                         <div class="status-icon"><i class="fa fa-search-plus"></i></div>
                     </a>
                     <a aria-label="Zoom out" class="roundbutton holder zoomer action" tabindex="100" id="command-zoom-out" href="#" title="Zoom out">
                         <div class="status-icon"><i class="fa fa-search-minus"></i></div>
                     </a>
                 </div>
-                <div id="small-icons" class="vbox">
+                <div id="small-icons-right" class="vbox small-icons">
```

Comment:
There is also an entry in python-main.js that uses the old id:

https://github.com/bbcmicrobit/PythonEditor/blob/0ce5e3f58ad837ad9726fae3f4794cf47a1d49fe/python-main.js#L1757-L1761